pkg/query-service/app/logs/v3/query_builder.go

```diff
@@ -84,7 +84,21 @@ func getClickhouseColumnName(key v3.AttributeKey) string {
 		columnType := getClickhouseLogsColumnType(key.Type)
 		columnDataType := getClickhouseLogsColumnDataType(key.DataType)
 		clickhouseColumn = fmt.Sprintf("%s_%s_value[indexOf(%s_%s_key, '%s')]", columnType, columnDataType, columnType, columnDataType, key.Key)
+		return clickhouseColumn
 	}
+
+	// check if it is a static field
+	if key.Type == v3.AttributeKeyTypeUnspecified {
```

Comment:
What do you think about adding a new `AttributeKeyTypeStatic` type here?

Reply:
Definately it will be good addition, but can be scoped as different PR.